packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -34,7 +26,6 @@ const CLASS_WITHOUT_SUBMIT =
 export default class QuanticSearchBox extends LightningElement {
   labels = {
```

Comment:
I wonder if we could move this default to the quanticSearchBoxInput, keep the @api property here, but let it be null,
And if it's empty or missing, the searchboxinput would display it's label.search value.